Document ncurses is required

```
src/surface.c:4:10: fatal error: ncurses.h: No such file or directory
    4 | #include <ncurses.h>
      |          ^~~~~~~~~~~
compilation terminated.
```

On Debian, it requires for example the package `libncurses-dev`

--- a/README.md
+++ b/README.md
@@ -8,6 +8,10 @@ Viewer of 3D models in ASCII, written in C.
 
 ## Compile an run the program
 
+You need the following libraries:
+  - ncurses (e.g. `libncurses-dev` on Debian)
+  - argp if you don't use Linux (e.g. `argp-standalone`)
+
 Compile the program using the `make` command:
 
 ```

